siliconcompiler/toolscripts/ubuntu26/install-mlir.sh

```
@@ -0,0 +1,110 @@
+# LLVM 19 needs cmake >= 3.20; ubuntu20 ships 3.16, so the version is pinned
+# here for every ubuntu the same way install-opensta.sh does it.
+python3 -m venv .mlir --clear
+. .mlir/bin/activate
+python3 -m pip install cmake==3.31.6
+
+version=$(python3 "${src_path}/_tools.py" --tool mlir --field git-commit)
+
+git clone --depth=1 --branch "${version}" \
+    "$(python3 "${src_path}/_tools.py" --tool mlir --field git-url)" llvm-project
+
+if [ ! -z ${PREFIX} ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)